.gitignore

```diff
@@ -3,12 +3,19 @@
 .bundle
 build/*
 tools/chef/cookbooks/*
+tools/chef/site-cookbooks/**/cookbooks/*
 tools/chef/tmp/*
 tools/assets/*
 bin/*
 .DS_Store
 secret.key
 
+# Chef
+**/.kitchen/
+**/cucumber/Gemfile.lock
+tools/chef/site-cookbooks/*/Berksfile.lock
+tools/chef/.chef/*_encrypted_data_bag_secret
```

Comment:
automatic key selection when editing data bags

I've been keen on trying knife-zero and perhaps using chef-vault data bags instead